autoware_adapi_v1_msgs/system/srv/OperateMRM.srv

```diff
@@ -0,0 +1,4 @@
+builtin_interfaces/Time stamp
+bool operate
```

Comment:
What does `operate` mean? Could you link the specification if it's written in the documentation?
cc @isamu-takagi

Reply:
It was planned to provide only the state. Would you like to add this API? @mkuri 
https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-interfaces/ad-api/list/api/fail_safe/

And is this service for forcing to trigger MRM by applications? If so, what behavior of MRM is selected?